### ISSUE TITLE
How to use `getRouteBaseName()` in Vuex action?

### ISSUE THREAD

**Issue by Bigdragon13th** [nuxt-modules/i18n#1012]
I have an action in Vuex that will behave differently based on current route that user is on. I try to use `getRouteBaseName` function to get the current route name without locale suffix to check for the route in said action, but can't find any way to do that. So the question is, how can I do that? Or should I change the way I implement the action?

**Comment by rchl** [comment]
If you have Nuxt Context available in your action (and you should be able to pass it if not) then the `getRouteBaseName` is available from `context.app.getRouteBaseName`.

**Comment by Bigdragon13th** [comment]
@rchl After I play around more, I found something weird. I've tried `this.app.getRouteBaseName` inside action (`this` is Vue instance), but for some reason it's undefined, while the methods that extend VueI18n like `getLocaleCookie` is available on `this.app.i18n.getLocaleCookie`

I don't want to pass nuxt context to action via payload because that means I've to pass it everywhere I call this action. But maybe I have to do that if no other way (I actually doing that right now).
